articles/go-sdk.html

```diff
@@ -207,9 +207,9 @@ <h3>View Go Documentations</h3>
 </p>
 
 <p>
-Please notes, currently (v1.10), if the <code>GOROOT</code> environment variable is not configured,
-we must specify the <code>goroot</code> flag when running the local godoc server.
-For example, <code>godoc -http=:9999 -goroot path/to/go/sdk</code>.
+Please note that currently (v1.10), if the <code>GOROOT</code> environment variable is set,

```

Comment:
missed a "not" here?

Reply:
I had '... variable is unset' on my mind - but got distracted.